pom.xml

```diff
@@ -181,6 +181,9 @@
     <!-- Remove hibermate-jpamodelgen prop when we move to jboss-integration-platform-bom 6.0.0.CR27+ -->
     <!-- When we move to hibernate 4.3+: version.org.hibernate.jpamodelgen should be removed and version.org.hibernate used instead  -->
     <version.org.hibernate.jpamodelgen>1.3.0.Final</version.org.hibernate.jpamodelgen>
+
+    <!-- Set default argLine value to empty to prevent surefire failure when jacoco is disabled -->
+    <argLine></argLine>
```

Comment:
Maybe we could use something like `surefire.argLine`? Or does it apply to other plugins as well?